### ISSUE TITLE
[Bug]: mo did not start successfully in five minutes

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#6427]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):sukki37 f271b53fa1455ca767a8eb4fd3a7b8d166732233
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

https://github.com/matrixorigin/matrixone/actions/runs/3424950677/jobs/5705235934
[logs_50532.zip](https://github.com/matrixorigin/matrixone/files/9967083/logs_50532.zip)

**Comment by w-zr** [comment]
Need to collect more logs on this issue.

**Comment by sukki37** [comment]
Repro on another commit 35f9ebcb, following is the log:
[init_timeout.log](https://github.com/matrixorigin/matrixone/files/9972825/init_timeout.log)

**Comment by w-zr** [comment]
> Repro on another commit [35f9ebc](https://github.com/matrixorigin/matrixone/commit/35f9ebcb29ed8f5bf7bd78634046d14340474595), following is the log: [init_timeout.log](https://github.com/matrixorigin/matrixone/files/9972825/init_timeout.log)

This log contains many `CommitTxn: storage commit failed. error:ExpectedDup` when run `create database if not exists mo_task`.

**Comment by w-zr** [comment]
This is probably related to #6532. Need more logs.

**Comment by w-zr** [comment]
> @w-zr What more log do you need? How do you plan to get it? Adding log in code? Different config?

I do not know how to reproduce it. @sukki37 gave me some logs. I found that they were related with #6532.
I'm trying to reproduce it to find out whether there are other reasons.

**Comment by w-zr** [comment]
Some integration tests may not complete in 5 min occasionally.
But this may be not related with this issue.
Now analyzing it.

**Comment by w-zr** [comment]
Above issue in integration tests is resolved by #6785.
Still trying to reproduce this issue.

**Comment by w-zr** [comment]
Still trying to reproduce this issue.

**Comment by aressu1985** [comment]
fixed: eebc6e93c952aafa5aac6ce46866f0eb3d38d7c5
